204 No Content response causes "fetch failed"

I have a keyboard handler in body
```
<body _="on keypress fetch `/kbd/${event.key}` with method:'POST'">
 ...
```

To send keystrokes to backend, but the backend just accepts it without sending anything back (this is a simple game, the actual contents is polled elsewhere).

my console says 
```
_hyperscript.min.js:19 fetch failed: 204 No Content (/kbd/d)

_hyperscript.min.js:19 Error: fetch failed: 204 No Content (/kbd/d)
    at _hyperscript.min.js:19:54868
```
(version 0.9.91)

The docs say "fetch throws on non-2xx responses (404, 500, etc.)," so I expect this should not throw.

--- a/src/core/config.js
+++ b/src/core/config.js
@@ -4,7 +4,7 @@ export const config = {
     attributes: "_, script, data-script",
     defaultTransition: "all 500ms ease-in",
     disableSelector: "[disable-scripting], [data-disable-scripting]",
-    fetchThrowsOn: [/4.*/, /5.*/],
+    fetchThrowsOn: [/^4/, /^5/],
     hideShowStrategies: {},
     logAll: false,
     mutatingMethods: {

--- a/test/commands/fetch.js
+++ b/test/commands/fetch.js
@@ -196,6 +196,24 @@ test.describe("the fetch command", () => {
 		await expect(find('div')).toHaveText("the body");
 	});
 
+	test("does not throw on 204 No Content", async ({html, find, page}) => {
+		await page.route('**/test', async (route) => {
+			await route.fulfill({ status: 204, body: '' });
+		});
+		await html("<div _='on click fetch /test then put \"ok\" into me'></div>");
+		await find('div').dispatchEvent('click');
+		await expect(find('div')).toHaveText("ok");
+	});
+
+	test("does not throw on 304 Not Modified", async ({html, find, page}) => {
+		await page.route('**/test', async (route) => {
+			await route.fulfill({ status: 304, body: '' });
+		});
+		await html("<div _='on click fetch /test then put \"ok\" into me'></div>");
+		await find('div').dispatchEvent('click');
+		await expect(find('div')).toHaveText("ok");
+	});
+
 	test("as response does not throw on 404", async ({html, find, page}) => {
 		await page.route('**/test', async (route) => {
 			await route.fulfill({ status: 404, body: 'not found' });

